packages/shared/lib/core/profile/actions/active-profile/login.ts

```diff
@@ -119,7 +119,7 @@ export async function login(loginOptions?: ILoginOptions): Promise<void> {
     } catch (err) {
         handleError(err)
         if (!loginOptions?.isFromOnboardingFlow) {
-            await logout()
+            await logout(true)
```

Comment:
it seems that we always want to `clearActiveProfile`. Maybe we should set its default to `true` or fully remove the param

Reply:
Ahh I see that it was added to idle will comment because we don't want it cleared on auto logout. Perhaps another place to make sure it is cleared is when we press back on the pin input screen.

Reply:
We don't actually want to clear the active profile here either? Because it has already been created so if there is an error logging in we want them to go back to the login screen from where they came and this requires the active profile.